modules/nf-core/kraken2/buildstandard/main.nf

```diff
@@ -0,0 +1,44 @@
+process KRAKEN2_BUILDSTANDARD {
+    label 'process_high'
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-8706a1dd73c6cc426e12dd4dd33a5e917b3989ae:c8cbdc8ff4101e6745f8ede6eb5261ef98bdaff4-0':
+        'biocontainers/mulled-v2-8706a1dd73c6cc426e12dd4dd33a5e917b3989ae:c8cbdc8ff4101e6745f8ede6eb5261ef98bdaff4-0' }"
+
+    input:
+
+    output:
+    path("$prefix"),     emit: db
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "kraken2_standard_db"
+    """
+    kraken2-build \\
+        --standard \\
+        $args \\
+        --threads ${task.cpus} \\
+        --db ${prefix}
+    kraken2-build --clean --db ${prefix}
```

Comment:
Technically this should also have an `args2`, although I don't think `--clean` accepts any other parameters?

Reply:
I don't think --clean has any parameters either. The reason I got rid of the clean option was my assumption is that if people are using the build-standard (rather than manually building or downloading a prebuilt index from [the index website](https://benlangmead.github.io/aws-indexes/k2) then they probably are not too interested in the particulars of how Kraken works and therefore are less likely to inspect the inside. As a result, I just made use of this as barebones as possible with no input.

Reply:
Fair enough, I still generally prefer not to make assumptions of what users want to do here (humans are weird)..., but I won't die on that hill :)